ui/panels-plugin/src/model/thresholds.ts

```diff
@@ -11,53 +11,38 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { UnitOptions } from '@perses-dev/components';
+import { UnitOptions, ThresholdColorPalette } from '@perses-dev/components';
+import { StepOptions, ThresholdOptions } from '@perses-dev/plugin-system';
 import { zip } from 'lodash-es';
 
-// TODO (sjcobb): pull threshold colors from perses charts theme
-export const ThresholdColors = {
-  GREEN: 'rgba(47, 191, 114, 1)', // green.500
-  YELLOW: 'rgba(255, 193, 7, 1)',
-  ORANGE: 'rgba(255, 159, 28, 0.9)', // orange.500
-  RED: 'rgba(234, 71, 71, 1)', // red.500
-};
-
-export const ThresholdColorsPalette = [ThresholdColors.ORANGE, ThresholdColors.RED];
```

Comment:
I really like that threshold colors moved to the chartsTheme, but one difference I'm seeing is previously orange and red were the defaults if a step color wasn't set explicitly, now it shows as blue for ex: 

<img width="715" alt="image" src="https://user-images.githubusercontent.com/9369625/220410240-af4635eb-caa4-463e-8d70-a6e027b33e70.png">